CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ if (MSVC)
 	 set(MSVC_TOOLSET_VERSION 142)
   endif()
   message(STATUS "MSVC_VERSION = ${MSVC_VERSION}, MSVC_TOOLSET_VERSION = ${MSVC_TOOLSET_VERSION}")
-  
```

Comment:
My editor configuration removes trailing white spaces on save -- I can update to not do that if desired.